src/Entity/Candidate.php

```diff
@@ -410,8 +410,8 @@ public function removePostulation(Postulation $postulation): static
         return $this;
     }
 
-    public function hasPostulated(JobOffer $jobOffer): bool
+    public function hasPostulated(?Postulation $postulation): ?bool
     {
-        return in_array($jobOffer, $this->getPostulations()->toArray());
+        return in_array($postulation, $this->getPostulations()->toArray());
```

Comment:
la bonne syntaxe est plutot `return $this->getPostulations()->exists($postulation)`  exists() est une méthode de Collection, ça évite de convertir la collection en tableau pour faire un in_array

Reply:
Ca ne veut pas fonctionner avec exists() : Expected type Closure, found App\Entity\Postulation

Reply:
my bad, j'ai confondu, c'est contains() la bonne méthode, et pas exists()